lib/imagekitio/sdk/version.rb

```diff
@@ -1,5 +1,5 @@
 module ImageKitIo
   module Sdk
-    VERSION = '2.3.0'
+    VERSION = '2.3.1'
```

Comment:
Given we are removing a lot of transformation parameter, we need a major version bump.